api_summary.html

```diff
@@ -108,81 +108,81 @@ <h2><a class="toc-backref" href="#id3">Load and run a model</a><a class="headerl
 in a single place.</p>
 <dl class="class">
 <dt id="onnxruntime.ModelMetadata">
-<em class="property">class </em><code class="descclassname">onnxruntime.</code><code class="descname">ModelMetadata</code><a class="headerlink" href="#onnxruntime.ModelMetadata" title="Permalink to this definition">¶</a></dt>
+<em class="property">class </em><code class="sig-prename descclassname">onnxruntime.</code><code class="sig-name descname">ModelMetadata</code><a class="headerlink" href="#onnxruntime.ModelMetadata" title="Permalink to this definition">¶</a></dt>
 <dd><p>Pre-defined and custom metadata about the model.
 It is usually used to identify the model used to run the prediction and
 facilitate the comparison.</p>
-<dl class="attribute">
+<dl class="method">
 <dt id="onnxruntime.ModelMetadata.custom_metadata_map">
-<code class="descname">custom_metadata_map</code><a class="headerlink" href="#onnxruntime.ModelMetadata.custom_metadata_map" title="Permalink to this definition">¶</a></dt>
+<em class="property">property </em><code class="sig-name descname">custom_metadata_map</code><a class="headerlink" href="#onnxruntime.ModelMetadata.custom_metadata_map" title="Permalink to this definition">¶</a></dt>
 <dd><p>additional metadata</p>
 </dd></dl>
 
-<dl class="attribute">
+<dl class="method">
 <dt id="onnxruntime.ModelMetadata.description">
-<code class="descname">description</code><a class="headerlink" href="#onnxruntime.ModelMetadata.description" title="Permalink to this definition">¶</a></dt>
+<em class="property">property </em><code class="sig-name descname">description</code><a class="headerlink" href="#onnxruntime.ModelMetadata.description" title="Permalink to this definition">¶</a></dt>
 <dd><p>description of the model</p>
 </dd></dl>
 
-<dl class="attribute">
```

Comment:
See a lot of changes making attribute -> method. Are these valid changes ?

Reply:
these seem like html changes. as long as the page renders, should be fine.

Reply:
Oh I see. thanks.